packages/itwinui-css/src/backdrop/backdrop.scss

```diff
@@ -5,12 +5,11 @@
 @mixin iui-backdrop {
   z-index: 998;
   isolation: isolate;
-  position: absolute;
+  position: fixed;
```

Comment:
I am not sure if `position: fixed` is the best way to go about this, because then backdrop will only work for full-screen elements

Reply:
I added a backdrop to the container dialog example, and it seems to work fine there.
![image](https://user-images.githubusercontent.com/37936169/184926646-e11d18c8-c01f-474a-aa84-2c6ea473a145.png)
What case are you thinking of?

Reply:
it works with the dialog wrapper because of this line:
https://github.com/iTwin/iTwinUI/blob/3b9b327ed8cf84db87dc1d61d356445cd5b051ba/packages/itwinui-css/src/dialog/dialog.scss#L16

but backdrop is meant to be a more generic component that will not necessarily be used with dialog

Reply:
So I made a new example with only the backdrop inside a div. Is this what you mean? Even with `position: absolute` the backdrop already covers the full page, not the space inside the div.

![image](https://user-images.githubusercontent.com/37936169/185229248-e31377c4-1712-4270-ae9a-3d5b4970b374.png)

Reply:
Maybe we need some modifier? If it is full-screen then it should be `position: fixed`, if not then `position: absolute`

Reply:
I added:
![image](https://user-images.githubusercontent.com/37936169/186003073-24ff0ee5-936c-44ed-9e83-0bf3b533d084.png)

I also considered making the acceptable options for `$position` either `fixed` or `absolute` and writing it like `position: #{$position}`. That way is a lot more succinct programming wise, but the current way is more clear why there's two different options. I can change it to the other way, or another way, if you guys think that'd be better.

Reply:
I think it will be enough to have just a simple modifier that overrides position. Because now you create two backdrop styles that only differ in position but has all the same properties like background color, transition...